doc/userguide/rules/tls-keywords.rst

```diff
@@ -153,11 +153,12 @@ Example::
 tls.version
 -----------
 
-Match on negotiated TLS/SSL version.
+Match on negotiated TLS/SSL version. Supports negation of the version match.
 
-Supported values: "1.0", "1.1", "1.2", "1.3"
+Supported values: "1.0", "1.1", "1.2", "1.3", "!1.0", "!1.1", "!1.2", "!1.3"
```

Comment:
Does this support multiple values ?
Like either 1.1 or 1.2 ?

Reply:
Yes. It will match on values 1.0->1.3, and !1.0->!1.3

Reply:
That was not what I was asking.
Is it possible with this keyword to match on TLS flows which will either be TLS 1.1 or TLS 1.2 (but not TLS 1.3, not TLS 1.0 and not SSLv2 or 3) ?

Reply:
I also found out recently about `SIGMATCH_HANDLE_NEGATION` 
Is this related ?

Reply:
Ah alright, my bad. After a bit of testing with suricata-verify, it seems that the answer may be no...
`SIGMATCH_HANDLE_NEGATION` appears to be very similar to the approach implemented here, it may be related - I am only aware of this now that you have mentioned it.

Reply:
friendly ping @Fredamabob  are you planning to work in this ?

Reply:
@catenacyber Hi, sorry about the late reply. I can look into this if you'd like, but it might take awhile for me to get back to it.

Reply:
Ok, no rush, just wanted to know if this PR still alive :-)

Reply:
Sorry for the long delay. I have updated it to use the flag SIGMATCH_HANDLE_NEGATION instead of using a new flag defined in detect-tls-version.h.

Reply:
Thanks for this improvement, I think CI will fail because of the PCRE1/PCRE2 thing cf other comment. Could you please take care of that ?